moviebot/controller/http_data_formatter.py

```diff
@@ -0,0 +1,132 @@
+"""This file contains a Messenger class to format data for HTTP requests."""
```

Comment:
Whats it meant by "Messenger class"?

Reply:
I forgot to remove it, thanks for catching that.